src/main/java/sirius/biz/storage/Storage.java

```diff
@@ -439,6 +440,23 @@ public OutputStream updateFile(@Nonnull StoredObject file) {
         return new UpdatingOutputStream(this, file);
     }
 
+    /**
+     * Creates a new output stream which updates the contents of the given file.
+     * <p>
+     * Note that most probably, the file will be updated once the stream is closed and not immediatelly on a write.
+     * Also note that it is essential to close the stream to release underlying resources.
+     * <p>
+     * After closing the stream, based on the result of writing the file, the complectionHandler is called.
+     *
+     * @param file              the file to update.
+     * @param completionHandler completion handler called once the file is written
+     * @return an output stream to write the contents to.
+     */
+    public OutputStream updateFileWithCompletionHandling(@Nonnull StoredObject file,
```

Comment:
isn't close on the outputstream called on the caller of this method - which also obviously has the StoredObject at hands? so when tell the caller that he called close?